src/app/lib/views/torrent_collection.js

```diff
@@ -76,179 +107,193 @@
                 return;
             }
 
+            if($('.search_in').is(':visible')) {
```

Comment:
`.togglesengines` and `.search_in` are used a lot, as well as the icons for them. there's a lot of room for typos here. can we make functions to get these components or constants for their selectors/class names?